questions/248/explanation.md

```diff
@@ -1,6 +1,6 @@
 `std::min()` and `std::max()` return references to their smallest/largest arguments, respectively. But what if they're equal?
 
-The C++17 standard says both for `std::min()` in §[alg.min.max]¶3  and for `std::max()` in §[alg.min.max]¶11:
+The C++23 standard says both for `std::min()` in §[alg.min.max]¶2  and for `std::max()` in §[alg.min.max]¶10:
```

Comment:
Maybe we should leave just "C++" here without specifying the exact version?

Reply:
Yeah let's do that as a rule. A little bit less work to port next time.